templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -179,43 +179,46 @@ icon. The second positional argument defines a list of categories for the
 integration.
 
 At this point, if you're following along and/or writing your own Hello World
-webhook, you have written enough code to test your integration.
+webhook, you have written enough code to test your integration. There are three
+tools which you can use to test your webhook - 2 command line tools and a GUI.
 
-First, get an API key from the Your bots section of your Zulip user's Settings
-page. If you haven't created a bot already, you can do that there. Then copy
-its API key and replace the placeholder `<api_key>` in the examples with
-your real key. This is how Zulip knows the request is from an authorized user.
+## Step 4: Manually testing the webhook
 
-Now you can test using Zulip itself, or curl on the command line.
+For either one of the command line tools, first, you'll need to get an API key
+from the **Your bots** section of your Zulip user's Settings page. If you
+haven't created a bot already, you can do that there. If you plan on using the
+GUI, make sure that the bot is of the "**Incoming Webhook**" type Then copy its API
+key and replace the placeholder `<api_key>` in the examples with your real key.
+This is how Zulip knows the request is from an authorized user.
 
-Using `manage.py` from within the Zulip development environment:
+### Tool 1: Curl
 
 ```
-(zulip-py3-venv)vagrant@vagrant-ubuntu-trusty-64:/srv/zulip$
-./manage.py send_webhook_fixture_message \
-    --fixture=zerver/webhooks/helloworld/fixtures/hello.json \
-    '--url=http://localhost:9991/api/v1/external/helloworld?api_key=<api_key>'
+curl -X POST -H "Content-Type: application/json" -d '{ "featured_title":"Marilyn Monroe", "featured_url":"https://en.wikipedia.org/wiki/Marilyn_Monroe" }' http://localhost:9991/api/v1/external/helloworld\?api_key\=<api_key>
 ```
-After which you should see something similar to:
+
+After which you should see:
```

Comment:
Just replace with `After running the above command, you should see something similar to:`

Reply:
Ok.